src/constants.h

```diff
@@ -104,6 +104,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // the file attempting to ensure a unique filename
 #define SCREENSHOT_MAX_SERIAL_TRIES 1000
 
+// Maximum speed (absolute value) for active objects.
+#define OBJECT_MAX_SPEED 500
```

Comment:
@nerzul: I investigated the maximum speed, and it turns out that there is an inefficiency in the collision detection code, that makes high-velocity objects require a disproportionate amount of memory and processing time. Both can easily increase quadratically with distance traveled in a single server step; worst case the increase is cubic.

Unless the collision detection code is optimized first, setting the maximum speed much higher than 500 may cause excessive computation time and/or memory use. In a 32-bit environment using an entity speed of 2000, and only a *single* entity, I consistently get huge delays and crashes due to `std::bad_alloc`...

The collision detection code does not seem trivial to fix, so for the moment, I propose leave that to be fixed at a later time, and keep the max speed as it is.